lib/inject.js

```diff
@@ -140,6 +138,25 @@ function injectPa11y(window, options, done) {
 		return false;
 	}
 
+	function hiddenAreasContainsElement(element, hiddenSelectors) {
```

Comment:
This function fails linting for its complexity, shall I split it up or just add the ignore comments?